server/locks_controller.go

```diff
@@ -57,8 +58,12 @@ func (l *LocksController) GetLock(w http.ResponseWriter, r *http.Request) {
 		LockedBy:        lock.Pull.Author,
 		Workspace:       lock.Workspace,
 		AtlantisVersion: l.AtlantisVersion,
+		CleanedBasePath: l.AtlantisURL.Path,
```

Comment:
Renamed this from AtlantisURL and am only passing the path around because think it's a best practice to only use paths in html so that it doesn't necessarily have to be served from the original domain.